lib/whitehall_importer/integrity_checker/body_check.rb

```diff
@@ -0,0 +1,29 @@
+module WhitehallImporter
+  class IntegrityChecker::BodyCheck
+    attr_reader :proposed_payload, :publishing_api_content
+
+    def initialize(proposed_payload, publishing_api_content)
+      @proposed_payload = proposed_payload
+      @publishing_api_content = publishing_api_content
```

Comment:
Rather than being passed in hashes that we have to pluck the body field from (with dig) can we pass in the body as strings? It'll save a lot of set-up in tests.

Reply:
Done.  This has made the tests much simpler to set up.